src/l10n/locale/pt.ts

```diff
@@ -1,3 +1,132 @@
 // Português
 
-export default {};
\ No newline at end of file
+export default {
+  //main.ts
+  "Open Dictionary View": "Abrir Vista do Dicionário",
```

Comment:
I have doubts about "Vista". In Portuguese we have "Janela" (as "Window") .
Generic software translate "View" as "Janela" (see attached image)
<img width="761" alt="image" src="https://user-images.githubusercontent.com/22862501/169617164-f2700d1e-7630-4475-b542-4277bdaefd89.png">